src/pages/_app.tsx

```diff
@@ -9,6 +9,15 @@ import mdxComponents from '@components/mdx'
 import defaultSeoConfig from 'src/next-seo.json'
 import {useURL} from '@hooks/useUrl'
 import '../styles/index.css'
+import Router from 'next/router'
+
+declare global {
+  interface Window {
+    ahoy: any
```

Comment:
I love modern development. We try so hard to hide from window, but the hacks to bring it back in just keep getting more interesting.